src/devices/HardwareMonitor/OpenHardwareMonitor.cs

```diff
@@ -169,29 +171,26 @@ public IList<Sensor> GetSensorList()
         {
             List<Sensor> ret = new List<Sensor>();
             ManagementObjectSearcher searcher = new ManagementObjectSearcher(@"root\OpenHardwareMonitor", "SELECT * FROM Sensor");
```

Comment:
I wonder if we could register devices in here

Reply:
What do you mean? You mean whether we could use a similar approach for registering our services?